resources/test-message-payloads/ahwp/ahwp-claim-next-steps.json

```diff
@@ -0,0 +1,26 @@
+// TEMPLATE NAME = uk.gov.fcp.ahwp.claim.review -------> DELETE THIS LINE BEFORE SENDING
+// UPDATE commsAddresses TO YOUR OWN EMAIL ADDRESS -------> DELETE THIS LINE BEFORE SENDING
```

Comment:
Throws an error as comments aren't allowed in JSON but these are just for reference/reminders.